week1/homework/src/server.js

```diff
@@ -9,7 +9,40 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    const reqUrl = request.url;
+    if (reqUrl === '/' || reqUrl === '/state') {
+      response.writeHead(200, { 'Content-Type': 'application/json' });
+      const home = {
```

Comment:
You don't have to write the response to a variable, since you only use it once, it makes more cense to write `response.write(JSON.stringify({'state': 'home'}));`